src/IO/external/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if (RADIUM_IO_ASSIMP)
     # For now, only configure in Debug mode to work with xcode
     ExternalProject_Add(assimp
             GIT_REPOSITORY https://github.com/assimp/assimp.git
-            GIT_TAG origin/master
+            GIT_TAG origin/assimp_5.0_release
```

Comment:
```suggestion
            GIT_TAG v5.0.1
```

Reply:
need to check the syntax, the idea is to use the release TAG instead of a branch HEAD

Reply:
I tried using the tag but the configuration step doesn't pass, neither with a commit tag, only with branches. Or maybe I didn't do it the right way.

Reply:
Ok, it works with this syntax, I must have messed it up last time